addons/mail/static/src/model/model_manager.js

```diff
@@ -997,12 +1031,25 @@ export class ModelManager {
      * @private
      * @param {Object} listener
      */
-    _markListenerToNotify(listener) {
+    _markListenerToNotify(listener, info) {
+        if (!(listener instanceof Listener)) {
+            throw new Error("Listener is not a listener", listener);
```

Comment:
2nd param of error is reserved for something else, this should be part of the string